erasure_code/gf_vect_mul_test.c

```diff
@@ -61,7 +61,10 @@ int main(int argc, char *argv[])
 	for (i = 0; i < TEST_SIZE; i++)
 		buff1[i] = rand();
 
-	gf_vect_mul(TEST_SIZE, gf_const_tbl, buff1, buff2);
+	if (gf_vect_mul(TEST_SIZE, gf_const_tbl, buff1, buff2) != 0) {
```

Comment:
The arm change is find but the test change has an issue.  We erroneously defined the base function as returning void instead of int like the multi-binary function gf_vect_mul().  This will cause systems running the base function to fail or "noarch" builds (`make -f Makefile.unx arch=noarch`).  Will have to also change the base function to return 0 and external header defn.

Reply:
Sg. Made the base function return 0 and changed the tests appropriately.